.gitignore

```
@@ -18,6 +18,17 @@ example/ionic-angular/www
 example/ionic-angular/.gradle
 example/ionic-angular/build
 example/ionic-angular/.ionic
+example/ionic-angular/.yalc
+
+example/ionic-angular-v2/dist
```

Would this work to include both samples?

```suggestion
example/ionic-angular*/dist
```